src/run_perf_test.py

```diff
@@ -36,6 +36,10 @@ def main():
     parser.add_argument("--stack", dest="stack", help="Stack name for performance test")
     parser.add_argument("--config", type=argparse.FileType("r"), help="Config file.", required=True)
     parser.add_argument("--keep", dest="keep", action="store_true", help="Do not delete the working temporary directory.")
+    parser.add_argument("--workload", default="nyc_taxis", help="The name of the workload from OpenSeach Benchmark Workloads")
+    parser.add_argument("--workload-options", default="{}", help="Json object with workload options to supply to OpenSearch Benchmark")
+    parser.add_argument("--warmup-iters", default=0, help="Number of warmup iterations for a workload before collecting data")
+    parser.add_argument("--test-iters", default=1, help="Number of test iterations for a workload")
```

Comment:
Are all of these arguments for mensor?

Reply:
Yes, I can update the help text to make that more clear

Reply:
Just a suggestion: Would it be better to say `Internal client param` since external users are not aware of what mensor it if they are looking at this code?

Reply:
Good point. I'm thinking `Internal client (Mensor) param`? That way internal users know how to get more information on these parameters.

Reply:
Sure sounds good. Thanks!